### PR TITLE
Add activeSpeakerPolicy and videoUplinkBandwidthPolicy in meeting manager config

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
- Add `activeSpeakerPolicy` and `videoUplinkBandwidthPolicy` in meeting manager config to allow builders to pass in 
  custom policies.
- Allow passing in config in `meetingManager.join` to allow more flexibility.
- Export MeetingManagerConfig object.

**Testing**
1. Have you successfully run `npm run build:release` locally? Yes

2. How did you test these changes? 
- Verified passing custom active speaker policy and video uplink bandwidth policy to the React meeting demo.

3. If you made changes to the component library, have you provided corresponding documentation changes? Yes.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/providers/MeetingProvider/docs/MeetingManager.stories.mdx [hunk 1]
```diff
@@ -43,7 +45,7 @@ Accepts a config object of `logLevel`, `postLogConfig`, `simulcastEnabled`, `ena
 Creates a meeting session using the meeting and attendee data passed. It will attempt to select default devices for the user.
 
 ```typescript
-import { EventReporter } from 'amazon-chime-sdk-js';
+import { EventReporter } from 'amazon-chime-sdk-js'; import MeetingManager from "./MeetingManager"; import {MeetingManagerConfig} from "./index";
```
Comment (devalevenkatesh) [inline src/providers/MeetingProvider/docs/MeetingManager.stories.mdx:48]: Could you please check if this renders correctly in storybook UI?

nit: We use single quotes for imports.
nit: We import directly from 'amazon-chime-sdk-component-library-react' and no relative imports.

File: src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx [hunk 2]
```diff
@@ -105,10 +105,51 @@ If you want to enable Amazon Voice Focus feature, you should enable Web Audio fo
 The `Logger` object that you want to be used in the meeting session.
 If you pass in a `Logger` object using this parameter, the `MeetingManager` will use this object instead of creating a logger based on `logLevel` and `postLogConfig` to initialize the meeting session.
 
-#### videoDownlinkBandwidthPolicy
+#### activeSpeakerPolicy
+
+The `ActiveSpeakerPolicy` object that you want to be used in the meeting session.
+For more information on `ActiveSpeakerPolicy`, check Amazon Chime JS SDK [ActiveSpeakerPolicy](https://aws.github.io/amazon-chime-sdk-js/interfaces/activespeakerpolicy.html).
+
+For example, if you do not want to priority the active speaker during a call you can override the
```
Comment (devalevenkatesh) [inline src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx:113]: Should we say "prioritize" instead of "priority"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,9 +12,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Added
+- Add `activeSpeakerPolicy` and `videoUplinkBandwidthPolicy` in meeting manager config to allow builders to pass in 
+  custom policies.
+- Allow passing in config in `meetingManager.join` to allow more flexibility.
```
Comment (devalevenkatesh) [inline CHANGELOG.md:17]: Should we re-word this to:
For more flexibility, allow passing `MeetingManagerConfig` to `meetingManager.join` method. Passing the config here would override config passed through `MeetingProvider` props.

File: CHANGELOG.md [hunk 4]
```diff
@@ -12,9 +12,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Added
+- Add `activeSpeakerPolicy` and `videoUplinkBandwidthPolicy` in meeting manager config to allow builders to pass in
```
Comment (devalevenkatesh) [inline CHANGELOG.md:15]: Suggest changing "meeting manager config" to actual type`MeetingManagerConfig`.

File: src/providers/MeetingProvider/MeetingManager.ts [hunk 5]
```diff
@@ -408,7 +418,7 @@ export class MeetingManager implements AudioVideoObserver {
     this.audioVideo?.setDeviceLabelTrigger(callback);
   }
 
-  setupActiveSpeakerDetection(): void {
+  setupActiveSpeakerDetection(activeSpeakerPolicy?: ActiveSpeakerPolicy): void {
```
[the remark this answers is not in the document]
Comment (ltrung) [reply src/providers/MeetingProvider/MeetingManager.ts]: Make sense. Updated.

File: src/providers/MeetingProvider/docs/MeetingManager.stories.mdx (hunk 1 above)
Comment (ltrung) [reply src/providers/MeetingProvider/docs/MeetingManager.stories.mdx]: Seem like my IDE auto insert theses :)

File: src/providers/MeetingProvider/types.ts [hunk 6]
```diff
@@ -16,6 +23,7 @@ export interface MeetingJoinData {
   attendeeInfo: any;
   deviceLabels?: DeviceLabels | DeviceLabelTrigger;
   eventReporter?: EventReporter;
+  managerConfig?: ManagerConfig;
```
Comment (devalevenkatesh) [inline src/providers/MeetingProvider/types.ts:26]: Should we change this to `meetingManagerConfig`, the value and not the type, I see we already made a mistake when earlier just the ManagerConfig type was exported 😓 ?

File: src/providers/MeetingProvider/MeetingManager.ts [hunk 7]
```diff
@@ -409,6 +418,8 @@ export class MeetingManager implements AudioVideoObserver {
   }
 
   setupActiveSpeakerDetection(): void {
+    const activeSpeakerPolicy  = this.meetingManagerConfig.activeSpeakerPolicy;
```
Comment (devalevenkatesh) [inline src/providers/MeetingProvider/MeetingManager.ts:421]: You can avoid 1 `activeSpeakerPolicy` by using de-structuring here :)

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**